Jenkinsfile

```diff
@@ -91,16 +96,26 @@ pipeline {
 
```

Comment:
Is it intended, that the coverage is not generated for the unit-tests? Or will this be an additional story?

Reply:
It is not implemented yet. I actually tried it but the unit tests had no coverage at all.
Nonetheless I will add it to the pr today.